python/examples/udp_client.py

```diff
@@ -1,48 +1,131 @@
 #!/usr/bin/env python3
 
+from datetime import datetime
 import os
+import math
 import socket
 import sys
+import time
 
 # Add the Python root directory (fusion-engine-client/python/) to the import search path to enable FusionEngine imports
 # if this application is being run directly out of the repository and is not installed as a pip package.
 root_dir = os.path.normpath(os.path.join(os.path.dirname(__file__), '..'))
 sys.path.insert(0, root_dir)
 
-from fusion_engine_client.messages.core import *
-
-from examples.manual_message_decode import decode_message
+from fusion_engine_client.parsers import FusionEngineDecoder
 from fusion_engine_client.utils.argument_parser import ArgumentParser
 
+from examples.message_decode import print_message
 
 if __name__ == "__main__":
     parser = ArgumentParser(description="""\
-Connect to a Point One device over UDP and print out the incoming message contents.
+Connect to a Point One device over UDP and print out the incoming message
+contents and/or log the messages to disk.
 
 When using UDP, you must configure the device to send data to your machine.
-
-This application assumes that the UDP stream contains only FusionEngine
-messages.
 """)
-    parser.add_argument('-p', '--port', type=int, default=12345,
-                        help="The UDP port to which messages are being sent.")
+
+    parser.add_argument('-f', '--format', default='p1log', choices=('p1log', 'raw', 'csv'),
+                        help="The format of the file to be generated when --output is enabled."
+                             "If 'p1log' (default), create a *.p1log file containing only FusionEngine messages."
```

Comment:
Suggest - for these to display as a list, otherwise they'll display as a paragraph instead of how they're broken up here